src/Sarif.Viewer.VisualStudio.Core/Models/SuppressionModel.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using System.Windows.Controls;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer.Controls;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.Sarif.Viewer.Models
+{
+    internal class SuppressionModel : NotifyPropertyChangedObject
+    {
+        private SuppressionKind _kind = SuppressionKind.External;
+        private SuppressionStatus _status = SuppressionStatus.Accepted;
+        private IEnumerable<SarifErrorListItem> _sarifErrorListItems;
+
+        public SuppressionModel(IEnumerable<SarifErrorListItem> sarifErrorListItems)
+        {
+            this._sarifErrorListItems = sarifErrorListItems;
+        }
```

Comment:
you could create another constructor that accepts only one sarifErrorListItem, with that, internally you could handle the way you want and update all the other places.